docs/source/containerization/kubernetes.mdx

```diff
@@ -29,7 +29,7 @@ router docker images.
 You can use helm to install charts from an OCI registry as follows:
 
 ```bash
-helm install --set router.configuration.telemetry.metrics.prometheus.enabled=true --set managedFederation.apiKey="REDACTED" --set managedFederation.graphRef="REDACTED" --create-namespace --namespace router-deploy router-test oci://ghcr.io/apollographql/helm-charts/router --version 0.15.1 --values router/values.yaml
+helm install --set router.configuration.telemetry.metrics.prometheus.enabled=true --set managedFederation.apiKey="REDACTED" --set managedFederation.graphRef="REDACTED" --create-namespace --namespace router-deploy router-test oci://ghcr.io/apollographql/helm-charts/router --version 0.16.0 --values router/values.yaml
```

Comment:
The version specified here is the version of the helm chart, which is moving at its own speed. This should be:
```suggestion
helm install --set router.configuration.telemetry.metrics.prometheus.enabled=true --set managedFederation.apiKey="REDACTED" --set managedFederation.graphRef="REDACTED" --create-namespace --namespace router-deploy router-test oci://ghcr.io/apollographql/helm-charts/router --version 0.1.20 --values router/values.yaml
```
We should update the notes on updating this to make it clear.
Also: see general comment about running `helm-docs` in the helm chart directory as part of this.

Reply:
While I agree it can be considered separately and evolve on its own, I also don't quite understand what the implications here would be of just coupling it to our releases and having the helm charts (too), published as part of a release that rallies around a single version bump.

Reply:
It's certainly lower cognitive overhead. In the past, we didn't publish the helm chart, so tying it to a release didn't make sense. It made more sense to let the version float as we were using it direct from main.

Now that we publish it, we probably do want to tie it with the version number of the release. I'm not sure what the consequence of winding the number back from `0.1.20` to `0.16.0` would be though... Oh well, we could just go ahead and do it since I don't think the number was correct on the last release.